eng/common/pipelines/templates/steps/verify-agent-os.yml

```diff
@@ -0,0 +1,35 @@
+# Template for all Python Scripts in this repository
+parameters: 
+  OSVmImage: ''
```

Comment:
To make this template a little easier to consume lets simply expect to find an OSVmImage variable defined and if it isn't we give an error saying that it should be set. That will allow us to consume this template by having one line:

```
- template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
```

Reply:
I really like this idea if it works.